test/example.css

```diff
@@ -2,6 +2,10 @@ html {
     box-sizing: border-box;
 }
 
+a {
+    animation: 100ms;
```

Comment:
This is testing [`time-min-milliseconds`](https://stylelint.io/user-guide/rules/time-min-milliseconds/) rule which has a minimum value of 100. If you change this to <100 it will fail, >100, is fine. I can remove this if you prefer.